docs/geedocs/5.2.5/answer/3481558.html

```diff
@@ -443,6 +443,16 @@ <h4><a name="geserveradmin_pdb"></a>Publish database</h4>
       <td>Optional</td>
       <td>Publish this database as the default one for the Earth Client to connect to if no database or virtual host is specified upon initial connection.</td>
     </tr>
+    <tr>
```

Comment:
I'd like to also see these options in the publishdb line above. It should look something like
```
--publishdb db_name --targetpath target_path [--vhname vh_name] [--setecdefault] [--enable_poisearch [--enable_enhancedsearch]]
```

Reply:
Added in the most recent commit.